apps/vscode/tsconfig.json

```diff
@@ -7,7 +7,6 @@
 			"ES2021"
 		],
 		"sourceMap": true,
-		"rootDir": "src",
```

Comment:
This change is to avoid the error:

```
File '/Users/juliasilge/Work/rstudio/quarto/apps/vscode/build.ts' is not under 'rootDir' '/Users/juliasilge/Work/rstudio/quarto/apps/vscode/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Matched by default include pattern '**/*'
```

I did some research on this problem, and I believe we don't need to specify a `rootDir` here anymore.

Reply:
I'm ok removing this so we get a clean problems pane. We can revert it if needed.